src/main/java/uk/gov/hmcts/reform/pip/data/management/controllers/PublicationController.java

```diff
@@ -355,6 +358,19 @@ public ResponseEntity<String> deleteArtefact(@RequestHeader("x-issuer-id") Strin
         return ResponseEntity.ok("Successfully deleted artefact: " + artefactId);
     }
 
+    @ApiResponses({
+        @ApiResponse(responseCode = OK_CODE, description = "Data Management - Artefact count per location - request "
+            + "accepted."),
+        @ApiResponse(responseCode = AUTH_ERROR_CODE, description = UNAUTHORIZED_DESCRIPTION)
+    })
+    @Operation(summary = "Return a count of artefacts per location")
+    @ResponseStatus(HttpStatus.OK)
+    @GetMapping("/count-by-location")
+    @IsAdmin
+    public ResponseEntity<String> countByLocation() {
```

Comment:
I'm thinking this should really return a model rather than a string - returning the model would mean the frontend wouldn't need to split / handle a CSV like string, and could then parse as a JSON object which will help debugging later down the line.

What do you think? I appreciate this would change the frontend parsing as well, so if you agree with the above I'm happy it being raised as a separate ticket on the backlog.

Reply:
After discussion with @ChrisS1512 , we've decided to move the above refactor into a separate ticket :)